src/integrationTest/java/com/mongodb/hibernate/query/select/BooleanExpressionWhereClauseIntegrationTests.java

```diff
@@ -54,9 +54,32 @@ void testBooleanFieldPathExpression(boolean negated) {
         assertSelectionQuery(
                 "from Book where" + (negated ? " not " : " ") + "outOfStock",
                 Book.class,
-                "{'aggregate': 'books', 'pipeline': [{'$match': {'outOfStock': {'$eq': "
-                        + (negated ? "false" : "true")
-                        + "}}}, {'$project': {'_id': true, 'discount': true, 'isbn13': true, 'outOfStock': true, 'price': true, 'publishYear': true, 'title': true}}]}",
+                """
+                {
+                  "aggregate": "books",
+                  "pipeline": [
+                    {
+                      "$match": {
+                        "outOfStock": {
+                          "$eq": %s
+                        }
+                      }
+                    },
+                    {
+                      "$project": {
+                        "_id": true,
+                        "discount": true,
+                        "isbn13": true,
+                        "outOfStock": true,
+                        "price": true,
+                        "publishYear": true,
+                        "title": true
+                      }
+                    }
+                  ]
+                }
+                """
+                        .formatted(negated ? "false" : "true"),
```

Comment:
In my last Sorting PR, I reformatted the expected JSON in all the other integration testing, but overlooked this class.